datasets/stroomstoringen/stroomstoringen.json

```diff
@@ -0,0 +1,148 @@
+{
+  "type": "dataset",
+  "id": "stroomstoringen",
+  "title": "Stroomstoringen",
+  "description": "Stroomstoringen (van vandaag) Liander netwerk regio Amsterdam.",
+  "dateCreated": "2021-06-14T00:00:00",
+  "dateModified": "2021-06-14T00:00:00",
+  "license": "Creative Commons, Naamsvermelding",
+  "homepage": "https://data.amsterdam.nl",
+  "language": "nl",
+  "status": "beschikbaar",
+  "version": "1.0.0",
+  "temporalUnit": "nvt",
+  "contactPoint": {
+    "name": "",
+    "email": "datadiensten.ois@amsterdam.nl"
+  },
+  "accrualPeriodicity": "dagelijks",
+  "spatialDescription": "Gemeente Amsterdam",
+  "theme": ["openbareruimte", "electra", "storingen"],
+  "publisher": "OIS",
+  "owner": "Gemeente Amsterdam, Openbareverlichting V&OR",
+  "authorizationGrantor": "Openbareverlichting V&OR",
+  "keywords": ["openbareruimte", "electra", "stroomstoringen"],
+  "hasBeginning": "2021-06-14T00:00:00",
+  "crs": "EPSG:28992",
+  "tables": [
+    {
+      "id": "stroomstoringen",
+      "type": "table",
+      "title": "Stroomstoringen",
+      "description": "Stroomstoringen (van vandaag) Liander netwerk regio Amsterdam",
+      "schema": {
+        "required": ["id"],
+        "mainGeometry": "geometrie",
+        "display": "id",
+        "$schema": "http://json-schema.org/draft-07/schema#",
+        "type": "object",
+        "additionalProperties": false,
+        "properties": {
+          "id": {
+            "type": "string",
+            "provenance": "objectid",
+            "description": "Uniek identificerend kenmerk stroomstoring."
+          },
+          "schema": {
+            "$ref": "https://schemas.data.amsterdam.nl/schema@1.1.1#/definitions/schema"
+          },
+          "geometry": {
+            "$ref": "https://geojson.org/schema/MultiPolygon.json",
+            "description": "Polygoon definitie stroomstoring."
+          },
+          "storingNummer": {
```

Comment:
Veldnaam is zonder 's', description met 's' (storing[s]nummer)

Reply:
Oeps